### PR TITLE
Update setup.cfg to add aliases for pyqt5 and pyside in extras_require

### PR DESCRIPTION
**Pull request by GenevieveBuckley**
# Description

This PR adds some aliases for the `extras_require` when installing napari. I sometimes find myself doing `pip install napari[pyqt]` which appears to install correctly, but because "pyqt" isn't an option for extras (should be "pyqt5" instead) I end up with no GUI backend installed.

It's not a big problem, just something that might be nice for convenience.

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation **I haven't done this, but I think that might be confusing. This is really to catch typos/abbreviations when someone types the install command from memory.**
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~

### CODE REVIEW

**Review by jni — APPROVED** [review]
I like this. I think there were some discussions about how this can be confusing because of the different versions (PySide is a different package from PySide2), but I don't think it's that terrible, since the original packages are not in widespread use anymore.

**Review by sofroniewn — APPROVED** [review]
Yeah, I'm fine with this too. I think the potential ambiguity does not out weigh the benefit, the other names relate to Qt4 which we've never started and we can decided what to do with our defaults etc. when Qt6 comes out and if we start supporting that.

Looks like we had some CI cancelations (too many concurrent runs?) so I've just triggered some manual reruns. I'll wait until they all pass and then merge.

Thanks for contribution @GenevieveBuckley!!

**Comment by sofroniewn** [comment]
This PR seems to have many more test runs than our other PRs, it's like the full matrix ran including windows 3.7 and windows 3.8 etc. Am I understanding that right? Is that expected @GenevieveBuckley or am I just confused. cc @jni?

**Comment by jni** [comment]
@sofroniewn I noticed that also and thought maybe something had changed in master, but I see now that it hasn't. 

If I look at the `.cirrus.yml`, I see that those checks are there conditional on `$CIRRUS_PR == ""`. Maybe something weird happened with Cirrus images during this one PR that has since been reverted? :shrug:

**Comment by jni** [comment]
Oh! Another hypothesis (I think probably correct): this PR comes from a branch on this repo, rather than from a fork. @GenevieveBuckley can you avoid doing that in the future? I think it is what GitHub defaults to in the web editor but there might be ways to avoid it.

**Pull request merged**
